### PR TITLE
[sem-048]:LABEL “ARTICULOS”

### PR DESCRIPTION
**Pull request by giovannicv**
Como usuario quiero ver un label indicando artículos.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -79,7 +79,7 @@
     "jsdom": "24.0.0",
     "postcss-html": "1.6.0",
     "postcss-scss": "4.0.9",
-    "prettier": "^3.2.5",
+    "prettier": "^3.3.0",
```
Comment (Ditmar) [inline package.json:82]: don't change the version of prettier

File: src/ui/components/atoms/label-articulo/Label.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,32 @@
+import type { StoryObj, Meta } from '@storybook/react';
+import Label from './Label';
+
+const meta = {
+  title: 'ui/components/atoms/Label',
```
Comment (Ditmar) [inline src/ui/components/atoms/label-articulo/Label.stories.tsx:5]: use the same name and structure of the path ui/components/atoms/label-articles

File: src/ui/components/atoms/label-articulo/styles.scss [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+    }
+    
+    @include respond-to('extra-large') {
+        width: 30%;
+    }
+}
+
+.label__text {
+    color: $ads-primary-main;
+    font-family: Montserrat;
```
Comment (Ditmar) [inline src/ui/components/atoms/label-articulo/styles.scss:25]: Nice please check if the font was correctly embedded in the project if it is not please do it

File: src/ui/components/atoms/label-articulo/styles.scss [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+    font-size: $ads-size-main;
+    font-weight: 800;
```
Comment (Ditmar) [inline src/ui/components/atoms/label-articulo/styles.scss:27]: this variable should be come from the globals styles

**Review by Ditmar — CHANGES_REQUESTED** [review]
review the comments

**Comment by DanX3N0** [comment]
FIGMA:
image

PR:
image

image

the colors are not the same as the figma.

**Comment by DanX3N0** [comment]
check the responsive 
image

**Review by Ditmar — APPROVED** [review]

**Pull request merged**